plugins.js

```diff
@@ -6,6 +6,20 @@ const Vision = require('vision');
 const HapiSwagger = require('hapi-swagger');
 const Good = require('good');
 const config = require('config');
+const appoloServer = require('apollo-server-hapi');
+
+// Imports the schema, resolvers and the model
```

Comment:
@Arjun-sna @rahulthampi We must find the best way to structure the folders and file for all schemas and resolvers. Currently shown for users alone

Reply:
Sure, we will define a better folder structure. Let's keep this PR's main focus on functionality. Once it is merged and we get to know what each file contains and how things work, it will be easy to give suggestions.

Reply:
@prasanna1211 are you planning to write the tests for this module. I would suggest to do it.